app/src/components/common/form/dropdown/index.tsx

```diff
@@ -361,10 +376,13 @@ export const Dropdown: React.FC<Props> = props => {
             {placeholder && !isDirty ? placeholder : activeItem.content}
             {!!activeItem.secondaryText && <SecondaryText>{activeItem.secondaryText}</SecondaryText>}
           </CurrentItem>
-          <ChevronWrapper>
-            <ChevronDown />
-            <ChevronUp />
-          </ChevronWrapper>
+          <DropdownButtonRight>
+            <CurrentItemExtra>{getItemExtraContent(currentItemIndex)}</CurrentItemExtra>
```

Comment:
I don't know if it is better for styling to have conditional rendered `<CurrentItemExtra>` in case of extraContent being empty, if is the case I would put after `activeItem` a declaration of `extraContent` that calls `getItemExtraContent(currentItemIndex)`.

Reply:
Doesn't seem to be any styling issue if `extraContent` is empty